src/components/tabs/js/tabsController.js

```diff
@@ -387,6 +387,7 @@ function MdTabsController ($scope, $element, $window, $mdConstant, $mdTabInkRipp
     $mdUtil.nextTick(function () {
       ctrl.updateInkBarStyles();
       updatePagination();
+      if(ctrl.stretchHeight) updateHeightFromParent();
```

Comment:
Add space for consistency please: `if(` -> `if (`

Reply:
@Splaktar addressed.